src/formatter.ts

```diff
@@ -413,6 +413,15 @@ export class Formatter {
 
   // Calculate the minimum width required to align and format `voices`.
   preCalculateMinTotalWidth(voices: Voice[]): number {
+    const unalignedPadding = Flow.DEFAULT_FONT_STACK[0].lookupMetric('stave.unalignedNotePadding');
```

Comment:
I think we should use the current font stack, no?

Reply:
um..yeah how do you do that?  I thought DEFAULT_FONT_STACK was the current one.

Reply:
Okay, yes, this is fine. I was thinking about element overrides with `setMusicFont`. This is good.